docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,106 @@
+# Getting started
+
+In this tutorial, you will set up a local version of our WordPress network.
+
+## Prerequisites
+
+Please ensure you have the following tools set up on your computer. Everything
+can be installed via [Homebrew](https://brew.sh/).
+
+### PHP
+
+We currently use the latest release on the 8.0.x branch, using commands such as:
+
+```bash
+$ brew install php@8.0
+```
+
+### Composer
+
+https://getcomposer.org/
+
+```bash
+$ brew install composer
+```
+
+The only global package currently needed is PHP CodeSniffer, and the WordPress
```

Comment:
I never did any of these global phpcs steps. Maybe we add that to an advanced setup doc and focus this on the core seeing as how I never did this and didn't notice for this long lol?

Reply:
I'm 47% sure my `composer security` still works as expected but I may need help confirming.

Reply:
Hmm... let me identify a step to confirm whether `composer security` does something. What happens now when you type that command?

Reply:
FWIW, this is what I see when I run the command at the moment:

```bash
% composer security
> phpcs --standard=phpcs.security.xml
```

Reply:
So, one way to confirm that Composer is successfully running the check:

1. Find any instance of `phpcs:disable` in the codebase, and remove that line. One example is this: https://github.com/MITLibraries/mitlib-wp-network/blob/master/web/app/themes/mitlib-child/functions.php#L214
2. With that line removed, there is now something for the security check to find. This should now be the output when running `composer security`

```bash
% composer security
> phpcs --standard=phpcs.security.xml

FILE: /Users/mjbernha/lando/mitlib-wp-network/web/app/themes/mitlib-child/functions.php
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 214 | ERROR | All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$output'.
     |       | (WordPress.Security.EscapeOutput.OutputNotEscaped)
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 1.39 secs; Memory: 10MB

Script phpcs --standard=phpcs.security.xml handling the security event returned with error code 1
```

Reply:
Yeah that works the same for me after removing the two lines from the child themes functions.php.

```
[I] ⋊> ~/p/mitlib-wp-network on hours  composer security                                                                                                
> phpcs --standard=phpcs.security.xml

FILE: /Users/jprevost/projects/mitlib-wp-network/web/app/themes/mitlib-child/functions.php
----------------------------------------------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------------------------------------------------------
 214 | ERROR | All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$output'.
     |       | (WordPress.Security.EscapeOutput.OutputNotEscaped)
----------------------------------------------------------------------------------------------------------------------------------------------------------------

Time: 2.35 secs; Memory: 10MB
```